KGP.TicketAPP.Utils/Helpers/JwtTokenHelper.cs

```diff
@@ -3,12 +3,14 @@
 using System.IdentityModel.Tokens.Jwt;
 using System.Security.Claims;
 using System.Text;
+using static KGP.TicketApp.Model.Database.Tables.User;
 
 namespace KGP.TicketApp.Backend.Helpers
 {
     public static class JwtTokenHelper
```

Comment:
zastanawia mnie, czy tego by się nie dało przekształcić w zwykłą klasę niestatyczną. Jest tu sporo metod statycznych, które jako pierwszy przyjmują argument string JwtToken. Taka klasa miałaby wewnątrz tę tekstową reprezentację (np property z public get i private set)

Reply:
Właśnie wydaje mi się, że klasa statyczna pasuje, ponieważ nie potrzebujemy więcej jej instancji. Ale jak chcesz to mogę zmienić te metody na metody rozszerzające typ string i będę je wywoływał na tych tokenach